### DISCUSSION TITLE
Some personal thoughts and suggestions.

### DISCUSSION THREAD

**Discussion by songjianet** [bcakmakoglu/vue-flow#427] [Q&A] [answered]
Hi Burak Cakmakoglu. I didn't find your contact details on your GitHub home page. For this reason I can only communicate with you in this way.
I am a front-end development engineer from China. I saw this project inadvertently. I use AntV/X6 to make workflows in the Apache DolphinScheduler project. In China, such projects are very popular. So, I want to lead some friends to do Chinese documentation support for this project, I wonder what you think about it.
At the same time, I also found some places where the project is not fully covered, which may be the reason for your limited energy, but I think we can optimize it together in the future, such as extending more mouse events, not just mouse double-click events, And it should also have the function of downloading PNG, SVG images, etc.
In terms of management, I think if a project wants to develop better, it should have its own organization, not in your own repository. You can create an organization and move it into the past.
Finally, I have an idea to use this work in the future Apache DolphinScheduler and Apache SeaTunnel, so I want to leave you a contact information. If it is convenient, you can send some of your commonly used contact information to my email, songjianet@apache .org.
Hope we have a good time working together and thank you for taking your time on this issue for me.

**Comment by bcakmakoglu** [comment, marked as answer]
Hey, thanks for opening up the discussion.
I'm happy to hear you're interested in the project.

So to answer your points.

1. Technically I am okay with supporting translations for the documentation. Shouldn't be too difficult but there wasn't any requests yet either :)
If you want to provide translations for chinese, I would be happy to add them to docs.

2. I am not sure what you mean? Basically all mouse events are supported, on nodes and edges, click, dblclick, mouse-enter/leave, ctx-menu etc. - What are you missing? You can browse the [TypeDocs](https://vueflow.dev/typedocs/) to get a more detailed insight on what is available and what isn't.

3. A screenshot feature can be implemented using [HTML-To-Image](https://www.npmjs.com/package/html-to-image). Right now, it's not built-in (and will not be built into the *core* of Vue Flow).
With the coming [Plugin API](https://github.com/bcakmakoglu/vue-flow/pull/363) I will add a simple screenshot plugin that would cover this use-case though.

4. I know how to create an organization 😅 
This has been a *personal* project of mine for over a year, hence it is on my *personal* name.
I don't see what benefit I would have in moving it into an organization (especially since organization could potentially even cost me money).
If you can enlighten me about why it would be better, I will consider it of course :)

5. About contacting me: Join the Vue Flow Discord and message me right there - I don't hand out e-mails on public forums 😄

**Comment by laterdayi** [comment]
@songjianet  @bcakmakoglu  Is there more to follow?
